src/librustc_resolve/macros.rs

```diff
@@ -328,7 +328,7 @@ impl<'a> Resolver<'a> {
                 if attr_kind == NonMacroAttrKind::Custom {
                     assert!(path.segments.len() == 1);
                     if !features.custom_attribute {
-                        let msg = format!("The attribute `{}` is currently unknown to the \
+                        let msg = format!("the attribute `{}` is currently unknown to the \
```

Comment:
With https://github.com/rust-lang/rust/pull/62086#issuecomment-509034287 this message is going to change entirely (only in `macros.rs`, not in `feature_gate.rs`), so we can avoid changing the tests one more time now.